stable/redis/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: redis
-version: 9.3.1
+version: 9.3.2
```

Comment:
Please bump minor version instead

Reply:
Updated in 22b6f2234b366fdcd708e92ff306afe064d73e97